.travis.yml

```diff
@@ -95,6 +95,18 @@ matrix:
         - cmake --build . --config $LRS_BUILD_CONFIG -- -j4
         - ls
 
+    - name: "ROS industrial-ci Kinetic"
```

Comment:
This CI job (and the one for named melodic too) does is, at the PR's state of the time of writing (20191107), to build the repo's code on ROS kinetic-compatible OS, which is by default Ubuntu 16.04. There's nothing specific to ROS (which is right as `librealsense` is independent from ROS, AFAIUC).

So maybe remove "ROS" from the job name, to avoid confusion.